README.rst

```diff
@@ -8,3 +8,121 @@ A GPL-licensed Python library that facilitates functional testing of quipucords.
    :target: https://travis-ci.org/quipucords/camayoc
 .. image:: https://codecov.io/gh/quipucords/camayoc/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/quipucords/camayoc
+
+Installation
+^^^^^^^^^^^^
+
+Camayoc only supports Python 3.5 and python 3.6, so it is recommended that you
+install Camayoc into a virtual environment. There are several tools available
+for managing virtual environments, below is an example of one way.::
+
+    git clone https://github.com/quipucords/camayoc.git
```

Comment:
I personally prefer to separate the cloning from the virtual environment and break it into steps 1-N.

You can provide a link to: 
- https://docs.python.org/3/tutorial/venv.html
- http://docs.python-guide.org/en/latest/dev/virtualenvs/

Step 1 clone repo:
<example>
Optional Steps (virtual environment):
<create & source environment>

Step 2 install dependencies
make install-dev
make install